monitor/remoteapi/server/server_test.go

```diff
@@ -335,7 +324,7 @@ func TestCreate(t *testing.T) {
 				Name:      "^^^^",
 				Cgroup:    "/trireme/123",
 				NS:        "/var/run/docker/netns/6f7287cc342b",
-				IPs:       map[string]string{"bridge": "172.17.0.1"},
+				IPs:       map[string]string{"bridge": "172.17.0.1", "ip": "thisisnotip"},
```

Comment:
You should add a unit test with utf8 characters. Creation of the PU should pass

Reply:
Added.